Update draft-irtf-nmrg-network-digital-twin-arch.md (#25)
cheneyzhoucheng/network-digital-twin: draft-irtf-nmrg-network-digital-twin-arch.md

As an example, the theory of bottleneck structures introduced in
{{G2-SIGCOMM}}, {{G2-SIGMETRICS}} can be used to construct a mathematical
model of the network (see also
{{?I-D.giraltyellamraju-alto-bsg-requirements}} for more info). A bottleneck
structure is a computational graph that efficiently captures the topology, the
routing and flow properties of the network. The graph embeds the latent
relationships that exist between bottlenecks and the application flows in a
distributed system, providing an efficient mathematical framework to compute
the ripple effects of perturbations (e.g., a flow arriving or departing from
the system, or the dynamic change in the capacity of a wireless link, among
others). Because these perturbations are mathematical derivatives of the
communication system, bottleneck structures can be used to compute optimized
network configurations, providing a natural engineering sandbox for building
network models. One of the key advantages of bottleneck structures is that
they can be used to compute (symbolically or numerically) key performance
indicators of the network (e.g., expected flow throughput, projected flow

> completion time, etc.) without using computationally intensive simulators.

Reviewer: ```suggestion
completion time) without using computationally intensive simulators.
```

This capability can be especially useful when building a digital twin or
a large-scale network, potentially saving orders or magnitude in computational
resources in comparison to simulation or emulation-based approaches.

The functional model aims to realize the dynamic evolution of network
performance evaluation and intelligent decision-making. Data-driven
AI/ML algorithms will play a great role in building complex network
functional models. As a research hotspot in recent years, many successful
cases have been demonstrated, such as {{RouteNet}}, {{MimicNet}}, etc.
In the future, in addition to improving the generalization ability and

> interpretability of AI models, we also need to focus on how to improve

Reviewer: ```suggestion
interpretability of AI models, there is also a need to focus on how to improve
```

  permission control to promote cooperation and information sharing
  among all parties.

*  Conflict Detection and Resolution: Identify and address conflicts,
  including user intents, access control policies, or multiple
  applications interacting within the digtial twin network system.
  Conflict detection and resolution techniques may use various mechanisms,
  such as rule-based policies, role-based access control, or dynamic conflict

>   resolution algorithms (e.g. {Pradeep2022}} and {{Zheng2022}}).

Reviewer: ```suggestion
  resolution algorithms (e.g., {Pradeep2022}} and {{Zheng2022}}).
```

> *  Energy-Efficient Twinning: Focus on energy efficiency in digital twin

Reviewer: ```suggestion
*  Energy-Efficient Twinning:
: Focus on energy efficiency in digital twin
```
